internal/report.go

```diff
@@ -240,51 +241,51 @@ type tableReportBody struct {
 func AnalyzeTables(conv *Conv, badWrites map[string]int64) (r []tableReport) {
 	// Process tables in alphabetical order. This ensures that tables
 	// appear in alphabetical order in report.txt.
-	var tables []string
-	for t := range conv.SrcSchema {
-		tables = append(tables, t)
-	}
-	sort.Strings(tables)
-	for _, srcTable := range tables {
-		if _, isPresent := conv.ToSpanner[srcTable]; isPresent {
-			r = append(r, buildTableReport(conv, srcTable, badWrites))
+	var tableIds []string
+	for tableId := range conv.SrcSchema {
+		tableIds = append(tableIds, tableId)
+	}
+	sort.Strings(tableIds)
```

Comment:
We keep sorting lists throughout this PR, can we do the following:

Option #1:
Use `TreeSet` DS for lists and `TreeHashMap` for maps. This preserves the ordering of the elements wrt the defined comparator. This will automatically keep the DS sorted.

Refs - 
1. https://pkg.go.dev/github.com/emirpasic/gods@v1.18.1/maps/treemap
2. https://pkg.go.dev/github.com/emirpasic/gods@v1.18.1/sets/treeset

Option #2:

1. Instead of sorting-on-demand, can we keep sort and store the source? This should be helpful for cases where the source is a `list`.
3. For cases where source is a `map`, we can use a `LinkedHashMap` to preserve the insertion order (note that we have to create the map in the sorted order for this to work).

Reply:
As discussed, in this phase we are not doing much changes in the conv struct. However this can be considered in the later phases of the project.